central/src/components/QuestionDashboard.tsx

```diff
@@ -84,44 +85,46 @@ export default function QuestionDashboard({
         </div>
       </>
     );
-
-  if (questions.length >= 1) {
+    if (questions.length >= 1) {
```

Comment:
here's that `InfiniteScroll` component that allows us to query for paginated results by `nextToken` via scroll